x-pack/test/functional/apps/spaces/enter_space.ts

```diff
@@ -57,7 +62,7 @@ export default function enterSpaceFunctonalTests({
 
       await PageObjects.spaceSelector.clickSpaceCard(spaceId);
 
-      await PageObjects.spaceSelector.expectHomePage(spaceId);
+      await PageObjects.spaceSelector.expectRoute('default', '/app/canvas');
```

Comment:
uiSettings service rejects invalid values on `write`, so we've got the old value unchanged.

Reply:
@legrego Can we 'patch' config document in tests? or change value in elasticsearch? what if we change value on es archiver data?

Reply:
Changing the value in the es archiver snapshot sounds reasonable to me. That's probably the most straightforward approach